_data/site.yaml

```diff
@@ -52,142 +52,53 @@ ga:
-      - name: We’re FedRAMP authorized
-        url: /docs/overview/fedramp-tracker/
-      - name: Applications we support
-        url: /docs/overview/portfolio-analysis/
-      - name: Benefits of PaaS
-        url: /docs/technology/responsibilities/
-      - name: Customer stories
-        url: /docs/customer-stories/epa-airnow-gov/
-      - name: Our team
-        url: /docs/overview/cloudgov-team/
-  - name: Get started
-    url: /sign-up/
-  - name: Pricing
-    key: pricing
-    children:
-      - name: Pricing
-        url: /pricing/
-      - name: Try a Free Sandbox
-        url: /docs/pricing/free-limited-sandbox/
-      - name: Quota Costs Billing & Limits
-        url: /docs/pricing/quotas/
-      - name: Support Packages
-        url: /support-packages/
-  - name: Documentation
-    key: docs
-    children:
-      - name: Deploying apps
-        url: /docs/deployment/deployment/
-      - name: Managing apps
-        url: /docs/management/plugins/
-      - name: Managing services
-        url: /docs/services/intro/
-      - name: Managing orgs
-        url: /docs/orgs-spaces/dashboard/
-  - name: News
-    url: /news/
-  - name: Knowledge base
-    url: /knowledge-base/
-  - name: Pages
-    url: /pages/
-  - name: Contact us
-    url: /contact/
+      - name: Navigation link
+        url: "#main-content"
+      - name: Navigation link
+        url: "#main-content"
+      - name: Navigation link
+        url: "#main-content"
 
 secondary_navigation:
-  - name: Platform Status
-    url: https://cloudgov.statuspage.io/
-  - name: Log in
-    url: https://login.fr.cloud.gov/login
-
-documentation_navigation:
-  - name: "Overview"
-    identifier: "overview"
-    icon: fa-home
-    weight: -150
-  - name: "Technology and security"
-    identifier: "technology"
-    icon: fa-lock
-    weight: -130
-  - name: "Customer stories"
-    identifier: "customer-stories"
-    icon: fa-star
-    weight: -120
-  - name: "Getting started"
-    identifier: "getting-started"
-    icon: fa-road
-    weight: -115
-  - name: "Deploying apps"
-    identifier: "deployment"
-    icon: fa-file-text
-    weight: -110
-  - name: "Services"
-    identifier: "services"
-    icon: fa-database
-    weight: -100
-  - name: "Managing apps"
-    identifier: "management"
-    icon: fa-rocket
-    weight: -100
-  - name: "Managing orgs"
-    identifier: "orgs-spaces"
-    icon: fa-users
-    weight: -100
-  - name: "Compliance"
-    identifier: "compliance"
-    icon: fa-check-square-o
-    weight: -95
-  - name: "Team documentation"
-    identifier: "ops"
-    icon: fa-wrench
+  - name: Secondary link
+    url: "#main-content"
+  - name: Another secondary link
+    url: "#main-content"
 
 # Search.gov configuration
 #
 # 1. Create an account with Search.gov https://search.usa.gov/signup
 # 2. Add a new site.
 # 3. Add your site/affiliate name here.
 searchgov:
-  endpoint: https://search.usa.gov # You should not change this.
```

Comment:
Don't delete the cloud.gov info

Reply:
This is only showing up because we reset the branch to 11ty.  The PR to migrate data should clean this up just fine.